digGenderExtractor/gender_extractor.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# @Author: ZwEin
+# @Date:   2016-06-21 12:36:47
+# @Last Modified by:   ZwEin
+# @Last Modified time: 2016-10-01 15:15:47
+
+import copy 
+import types
+from digExtractor.extractor import Extractor
+from digGenderExtractor.digge import DIGGE
+
+class GenderExtractor(Extractor):
+
+    def __init__(self):
+        self.renamed_input_fields = ['text']  # ? renamed_input_fields
+
+    def extract(self, doc):
+        return DIGGE.extract(doc['text'])
```

Comment:
Hi Jason, thanks for points out, 👍 . If 'text' filed is not exist, should I just ignore it or throw a exception? thank you